master/networking/migrate-pools.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Migrate from one IP pool to another
+---
+
+### Big picture
+
+Migrate pods from one IP pool to another on a running cluster without network disruption.
+
+### Value
+
+Pods are assigned IP addresses from IP pools that you configure in {{site.prodname}}. As pods increase, you may want pods to move a larger CIDR. Or, you may need to move pods from a CIDR that was used by mistake. {{site.prodname}} lets you migrate from one IP pool to another one on a running cluster without network disruption. 
+
+### Features
+
+This how-to guide uses the following Calico features:
+
+- **IPPool** resource 
+
+### Concepts
+
+#### IP pools and cluster CIDRs
+
+{{site.prodname}} supports changing IP pools for pods if the IP pool you are changing to is within the same cluster CIDR. Although it is technically feasible to change a pod to an IP pool outside the cluster CIDR, we do not recommend it; this requires restarting components (affecting application availability), and if you change to a CIDR range that doesn’t include the already-assigned value, existing pods will lose connectivity.
```

Comment:
I think this paragraph is close, but not quite accurate. 

I'd recommend adjusting to the following:

> {{site.prodname}} supports using multiple disjoint IP pool CIDRs within the cluster. However, Kubernetes expects that all pods have addresses within the same cluster CIDR. This means that while it is technically feasible to create an IP pool outside of the cluster CIDR, we do not recommend it. Pods allocated addresses outside of the Kubernetes cluster CIDR will lose network connectivity.

I'd probably get rid of the "it requries restarting components" bit - I think it just complicates the message we're trying to send.

Reply:
Fixed.

Reply:
Fixed.